playbooks/roles/edxapp/defaults/main.yml

```diff
@@ -1455,6 +1466,13 @@ lms_env_config:
   GOOGLE_SITE_VERIFICATION_ID: "{{ EDXAPP_GOOGLE_SITE_VERIFICATION_ID }}"
   STATIC_URL_BASE: "{{ EDXAPP_LMS_STATIC_URL_BASE }}"
   LMS_ALLOWED_HOSTS: "{{ EDXAPP_LMS_ALLOWED_HOSTS }}"
+  GET_SECRETS_FROM_AZURE_KEYVAULT: "{{ EDXAPP_GET_SECRETS_FROM_AZURE_KEYVAULT }}"
+  AZURE_MSI_REQUEST_URI: "{{ EDXAPP_AZURE_MSI_REQUEST_URI }}" 
+  KEYVAULT_PAYLOAD: "{{ EDXAPP_KEYVAULT_PAYLOAD }}"
+  KEYVAULT_URL": "{{ EDXAPP_KEYVAULT_URL }}"
+  LMS_NON_SECURE_KEY_NAME": "{{ EDXAPP_LMS_NON_SECURE_KEY_NAME }}"
+  LMS_SECURE_KEY_NAME": "{{ EDXAPP_LMS_SECURE_KEY_NAME }}"
+  API_VERSION": "{{ EDXAPP_API_VERSION }}"
```

Comment:
Suggest renaming as API_VERSION isn't specific enough.  Same comment for line 1523.

Reply:
updated